src/NServiceBus.Extensions.DependencyInjection.Autofac.AcceptanceTests/When_using_externally_managed_container.cs

```diff
@@ -0,0 +1,95 @@
+﻿namespace NServiceBus.Extensions.DependencyInjection.Autofac.AcceptanceTests
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using global::Autofac;
+    using global::Autofac.Extensions.DependencyInjection;
+    using Microsoft.Extensions.DependencyInjection;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_using_externally_managed_container : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_resolve_dependencies_from_serviceprovider()
+        {
+            IContainer container = null;
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<EndpointWithExternallyManagedContainer>(endpointBuilder =>
+                {
+                    var serviceCollection = new ServiceCollection();
+                    endpointBuilder.CustomConfig(endpointConfiguration => endpointConfiguration
```

Comment:
Should we add a component via the servicecollection as well?

Reply:
could make sense, I'll add that